pkg/component/observability/plutono/dashboards/seed/fluent-bit-dashboard.json

```diff
@@ -394,309 +605,453 @@
+      "title": "[Output Plugin] No Metadata Records",
+      "type": "stat"
+    },
+    {
+      "datasource": "prometheus-aggregate",
+      "description": "buffer sizes output plugin",
+      "fieldConfig": {
+        "defaults": {
+          "color": {
+            "mode": "palette-classic"
+          },
+          "custom": {
+            "axisLabel": "",
+            "axisPlacement": "auto",
+            "barAlignment": 0,
+            "drawStyle": "line",
+            "fillOpacity": 0,
+            "gradientMode": "none",
+            "hideFrom": {
+              "graph": false,
+              "legend": false,
+              "tooltip": false
+            },
+            "lineInterpolation": "linear",
+            "lineWidth": 1,
+            "pointSize": 5,
+            "scaleDistribution": {
+              "type": "linear"
+            },
+            "showPoints": "never",
+            "spanNulls": true
+          },
+          "mappings": [],
+          "noValue": "0",
+          "thresholds": {
+            "mode": "absolute",
+            "steps": [
+              {
+                "color": "green",
+                "value": null
+              }
+            ]
+          },
+          "unit": "short"
+        },
+        "overrides": []
       },
-      "type": "graph",
-      "xaxis": {
-        "buckets": null,
-        "mode": "time",
-        "name": null,
-        "show": true,
-        "values": []
+      "gridPos": {
+        "h": 5,
+        "w": 12,
+        "x": 12,
+        "y": 22
       },
-      "yaxes": [
-        {
-          "$$hashKey": "object:421",
-          "format": "short",
-          "label": "",
-          "logBase": 1,
-          "max": null,
-          "min": "0",
-          "show": true
+      "id": 95,
+      "options": {
+        "legend": {
+          "calcs": [
+            "mean",
+            "last"
+          ],
+          "displayMode": "table",
+          "placement": "right"
         },
+        "tooltipOptions": {
+          "mode": "single"
+        }
+      },
+      "pluginVersion": "7.5.42",
+      "targets": [
         {
-          "$$hashKey": "object:422",
-          "format": "short",
-          "label": "",
-          "logBase": 1,
-          "max": null,
-          "min": null,
-          "show": true
+          "exemplar": true,
+          "expr": "fluentbit_vali_gardener_dque_size{pod=~\"$pod\",name=~\"$host\"}",
+          "instant": false,
+          "interval": "",
+          "legendFormat": "{{pod}}/{{name}}",
+          "queryType": "randomWalk",
+          "refId": "A"
         }
       ],
-      "yaxis": {
-        "align": false,
-        "alignLevel": null
-      }
+      "title": "[Output Plugin] DQue Current Size",
+      "type": "timeseries"
     },
     {
-      "datasource": null,
-      "description": "Table which shows the total incoming logs from the Output plugin",
+      "datasource": "prometheus-aggregate",
+      "description": "Output plugin total logs",
```

Comment:
This description doesn't really match the title which is "[Output Plugin] Incoming Logs Rate", and the query